src/server/chunk_server/chunk_server_lease_service_impl.cc

```diff
@@ -15,18 +21,94 @@ namespace service {
 grpc::Status ChunkServerLeaseServiceImpl::GrantLease(
     ServerContext* context, const GrantLeaseRequest* request,
     GrantLeaseReply* reply) {
-  // TODO(someone): implement the GFS chunk server logic here
+  LOG(INFO) << "Received GrantLeaseRequest:" << (*request).DebugString();
   *reply->mutable_request() = *request;
-  reply->set_status(GrantLeaseReply::ACCEPTED);
-  return grpc::Status::OK;
+
+  LOG(INFO) << "Preparing redo logs for GrantLease of file handle: "
+            << request->chunk_handle();
+  // TODO: may not be necessary depending on how crash-resistent we want to be
```

Comment:
nit: This can just be comments/todos here, no need to log what weren't doing, to avoid noise in our log. Thoughts?

Reply:
make it look impressive :)

Reply:
Haha, cool.